### ISSUE TITLE
Take ownership of the XSOAR connector 

### ISSUE THREAD

**Issue by nino-filigran** [OpenCTI-Platform/connectors#2073]
## Use case

Palo Alto has stopped maintaining their integration since version 5.12 with new filter structure. Since then, the connector is broken. 

## Current Workaround

Create a taxii feed with XSOAR however the capabilities are not the same.

**Comment by annoyingapt** [comment]
Yeah the opencti integration in xsoar is rubbish. I'm using the graphql integration instead. It was painful to update everything when changes were made to filters.
